tutorial/output-cache-open-ai/README.md

```diff
@@ -0,0 +1,53 @@
+# .NET 8 Web Application with Redis Output Caching and Azure Open AI
+
+## Overview
+This repository contains sample code for an AI image generation web application. It calls into Azure Open AI DallE to generate images based on user prompt. There are two features the sample highlights:
+* It allows responses from the /cached/ endpoint to be saved in Azure Cache for Redis through the [IOutputCache() abstraction](https://learn.microsoft.com/aspnet/core/performance/caching/output?preserve-view=true&view=aspnetcore-8.0#cache-storage). 
+* It uses semantic caching to cache responses from similar prompts using the [Redis OM for .NET](https://github.com/redis/redis-om-dotnet)
+
+To experiment with the demo:
+1. call the /nocache/ endpoint with your command prompt. Refresh the browser. Observe that refresh takes a noticeable delay and a new picture will be generated.
```

Comment:
Could you provide an example command of how to `call the /nocache/` through command prompt? I would expect a url with `/noncache/` to load in the browser?

Reply:
fixed. will include in the next commit.

Reply:
Let us call out user need to update the url in browser to include the `nocache` endpoint